infra/Pos/Reporting/Methods.hs

```diff
@@ -118,23 +117,20 @@ ipv4Local w =
 
 -- | Retrieves node info that we would like to know when analyzing
 -- malicious behavior of node.
-getNodeInfo :: (MonadDHT m, MonadIO m) => m Text
+getNodeInfo :: (MonadIO m) => m Text
```

Comment:
This function is supposed to say something about neighbors, isn't it possible to do so without concrete implementation?